### PR TITLE
Update content on home page and redesign footer

### PR DESCRIPTION
**Pull request by Sharonsyra**
**What does this PR do?**

- It updates the home page content
- Updates footer section

**Description of tasks performed**

- It updates the home page content in the hero-banner, mission and testimonials section
- It aligns and changes the footer logo to greyscale
- It adds a caption to the footer

**Screenshots**

screen shot 2017-09-11 at 3 17 43 pm
screen shot 2017-09-11 at 3 17 54 pm
screen shot 2017-09-11 at 3 18 03 pm
screen shot 2017-09-11 at 3 18 11 pm
screen shot 2017-09-11 at 3 18 20 pm

### CODE REVIEW

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html [hunk 1]
```diff
@@ -7,11 +7,9 @@
...
-          <h1 class="intro-title">Improve your digital storytelling with a combination of online
-            webinars and face-to-face training in:</h1>
-          <p>Free courses on everything from search to social media, to help you grow your business
-            or career.</p>
-
+          <h1 class="intro-title">Free world-class digital journalism course with video explainers, concise tip-sheets and practical exercises to immediately improve your reporting and storytelling.
+          </h1>
+        
           <template is="dom-if" if="[[!currentUser.loggedIn]]">
             <a href="[[currentUser.loginUrl]]" class="waves-effect waves-light btn-large">Start
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/components/pages/homepage.html]: @Sharonsyra Indentation.

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html [hunk 2]
```diff
@@ -27,22 +25,10 @@ <h1 class="intro-title">Improve your digital storytelling with a combination of
...
-          <p>
-            The Academy will also offer online courses and webinars, designed to teach just one tool
-            or technique at a time, so participants can upgrade their skills at their own pace.
-          </p>
+          <h2>Learn on the Go</h2>
           <p>
-            The training will be spearheaded by the continent’s largest digital journalism network,
-            Code for Africa, with support from the Google News Lab and World Bank.
+          As a journalist living in the digital age, a huge demand falls on you to be up to date with the latest in data gathering and processing skills.These courses are tailored to help you do just that.
+          From learning Google apps that can make your investigative work easier to delving deeper into the intricacies of extracting and understanding data. You will become a head honcho in the latest techniques used in news gathering.
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/components/pages/homepage.html]: @Sharonsyra br/ ?

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html (hunk 1 above)
Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: On it, thanks.

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html (hunk 2 above)
Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: What do you mean?

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html [hunk 3]
```diff
@@ -182,47 +156,63 @@ <h2 class="section-label">Testimonials</h2>
...
               </div>
               <img src="/modules/explorer/_static/images/testimonial1.jpg" alt="testimonial image">
             </div>
             <div id="testimonial2" class="testimonial-div">
               <div class="testimonial-card">
                 <h3>Words to live by</h3>
-                <p>Adrian Simpson set up a basic website for his new speechwriting business and
-                  began
-                  advertising online through search.
+                <p>Adrian Simpson set up a basic website for his new speechwriting business and began advertising online through search.
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/components/pages/homepage.html]: @Sharonsyra is this the right content?

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html [hunk 4]
```diff
@@ -182,47 +156,63 @@ <h2 class="section-label">Testimonials</h2>
...
             </div>
           </div>
           <div class="card-content grey lighten-4">
             <div id="testimonial1" class="testimonial-div">
               <div class="testimonial-card">
                 <h3>Words to live by</h3>
-                <p>Adrian Simpson set up a basic website for his new speechwriting business and
-                  began
-                  advertising online through search.
+                <p>“StoryLab Academy has provided an avenue to learn courses not offered in my school. Given that I am a writer, courses like verification will enable me write better stories.”
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/components/pages/homepage.html]: @Sharonsyra how about &ldquo and &rdquo ?

File: coursebuilder/modules/explorer/templates/explorer.html [hunk 5]
```diff
@@ -52,17 +52,31 @@
     <div class="footer-bottom">
         <div class="row">
             <div class="col l8 m12 footer-appstores">
+                <div class="col s2">
                 <a target="_blank" href="https://codeforafrica.org/">
-                    <img class="store cfa-footer-logo" src="/modules/explorer/_static/images/c4a.png">
+                    <img style="" class="store cfa-footer-logo" src="/modules/explorer/_static/images/c4a.png">
```
Comment (chegejames) [inline coursebuilder/modules/explorer/templates/explorer.html]: @Sharonsyra empty in-line styling?

Comment (Sharonsyra) [reply coursebuilder/modules/explorer/templates/explorer.html]: Let me fix that now

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html (hunk 4 above)
Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: Good idea let me use that.

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html (hunk 3 above)
Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: Muthoni added new data that i was supposed to update. It is in the word document.

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html (hunk 2 above)
Comment (chegejames) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: Did you intend to have a line break between "just that." and 
"From learning" ?

File: coursebuilder/modules/explorer/templates/explorer.html [hunk 6]
```diff
@@ -52,17 +52,31 @@
...
                 </a>
+                </div>
 
+                <div class="col s3">
```
Comment (chegejames) [inline coursebuilder/modules/explorer/templates/explorer.html]: @Sharonsyra Indentation.

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html (hunk 2 above)
Comment (Sharonsyra) [reply coursebuilder/modules/explorer/_static/components/pages/homepage.html]: Aah! Thanks let me add one.

File: coursebuilder/modules/explorer/_static/components/pages/homepage.html [hunk 7]
```diff
@@ -182,47 +164,67 @@ <h2 class="section-label">Testimonials</h2>
...
           <div class="card-tabs">
             <div class="row">
-              <div class="testimonials-card-tabs">
-                <ul class="tabs tabs-fixed-width">
-                  <li class="tab"><a class="active" href="#testimonial1">adrian</a></li>
-                  <li class="tab"><a href="#testimonial2">rachal</a></li>
-                  <li class="tab"><a href="#testimonial3">mike</a></li>
-                </ul>
-              </div>
+            <div class="testimonials-card-tabs">
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/components/pages/homepage.html]: @Sharonsyra Please revert this until there is content for this section.

File: coursebuilder/modules/explorer/templates/explorer.html [hunk 8]
```diff
@@ -52,17 +52,31 @@
     <div class="footer-bottom">
         <div class="row">
             <div class="col l8 m12 footer-appstores">
+                <div class="col s2">
                 <a target="_blank" href="https://codeforafrica.org/">
```
Comment (chegejames) [inline coursebuilder/modules/explorer/templates/explorer.html]: @Sharonsyra You may replace these three logos with that in https://academy.codeforafrica.org/img/logos/powered-by-gray.png It does not very good on mobile as it is now. For consistency please refer to the footer in https://academy.codeforafrica.org/

File: frontend/index.html [hunk 9]
```diff
@@ -87,8 +87,8 @@
   <!--  Hero Banner  -->
```
Comment (chegejames) [inline frontend/index.html:87]: @Sharonsyra How is this page related to coursebuilder/modules/explorer/_static/components/pages/homepage.html?

Comment (Sharonsyra) [reply frontend/index.html:87]: I have updated it now.
